Resources/public/js/views/services/ez-roleserversideviewservice.js

```diff
@@ -41,16 +45,16 @@ YUI.add('ez-roleserversideviewservice', function (Y) {
                 service = this;
 
             this._assignRoleNotificationStarted(
-                data.roleId, data.roleName, e.selection
+                data.roleRestId, data.roleName, e.selection
```

Comment:
@yannickroger I renamed it to clarify, because there was a "roleId" before which actually was a role REST ID, and this fix introduces a role ID which is an actual role ID :) So I thought this is the cleanest way around the conflict, avoiding future confusion.